.tests/cypress/integration/mini-cart/add-to-cart-configurable.js

```diff
@@ -1,85 +1,82 @@
 describe('Configurable product', function () {
 
-    before(() => {
+  before(() => {
 
-        // Hide premissions popup
-        cy.setCookie('permission-cookies', 'true')
-        cy.setCookie('permission-profiling', 'true')
-        cy.setCookie('mage-cache-sessid', 'true')
-        // Keep cookies beween tests
-        Cypress.Cookies.defaults({
-            whitelist: [
-                'frontend',
-                'X-Magento-Vary',
-                'permission-profiling',
-                'PHPSESSID',
-                'form_key',
-            ]
-        })
+    // Hide premissions popup
+    cy.setCookie('permission-cookies', 'true')
+    cy.setCookie('permission-profiling', 'true')
+    cy.setCookie('mage-cache-sessid', 'true')
+    // Keep cookies beween tests
+    Cypress.Cookies.defaults({
+      whitelist: [
+        'frontend',
+        'X-Magento-Vary',
+        'permission-profiling',
+        'PHPSESSID',
+        'form_key',
+      ]
     })
+  })
 
-    it('Visits product', () => {
-        cy.visit('https://alpaca-ce-solr-demo.snowdog.pro/configurable-product')
-        cy.get('.breadcrumbs__list').should('be.visible')
-    })
+  it('Visits product', () => {
+    cy.visit('https://alpaca-ce-solr-demo.snowdog.pro/configurable-product')
```

Comment:
baseUrl has already been set in cypress.json file. Please correct this to `cy.visit('/configurable-product)`